source/type/data.cpp

```diff
@@ -434,6 +434,20 @@ static void build_datasets(navitia::type::VehicleJourney* vj) {
     }
 }
 
+/*
+ * Build other nodes than dataset.
+ *
+ * @param vj The vehicle journey to browse
+ */
+static void build_nodes(navitia::type::VehicleJourney* vj) {
```

Comment:
The name of this function doesn't mean anything

Reply:
Done